tests/run.py

```diff
@@ -2,41 +2,136 @@
 
 import sys
 import argparse
+import random
 
 from helper import Test, TestGroup, run, find_profiles
 from talent_options import talent_combinations
 
 FIGHT_STYLES = ('Patchwerk', 'DungeonSlice', 'HeavyMovement')
 COVENANTS = ('Kyrian', 'Venthyr', 'Night_Fae', 'Necrolord')
 
-parser = argparse.ArgumentParser(description='Run simc tests')
-parser.add_argument('specialization', metavar='spec', type=str,
-                    help='Simc specialization in the form of CLASS_SPEC, eg. Priest_Shadow')
-parser.add_argument('--test-trinkets', default=False, action='store_true',
-                    help='Test trinkets. Trinket tests require simc-support library to be installed.')
-parser.add_argument('--trinkets-fight-style', default='DungeonSlice', type=str,
-                    help='Fight style used for trinket simulations.')
-args = parser.parse_args()
 
-def test_trinkets(klass : str, path : str):
-    if not args.test_trinkets:
+def test_talents(klass: str, path: str):
+    for fight_style in FIGHT_STYLES:
+        grp = TestGroup('{}/{}/talents'.format(profile, fight_style),
+                        fight_style=fight_style, profile=path)
+        tests.append(grp)
+        for talents in talent_combinations(klass):
+            Test(talents, group=grp, args=[('talents', talents)])
+
+
+def test_covenants(klass: str, path: str):
+    try:
+        from simc_support.game_data.WowSpec import get_wow_spec_from_combined_simc_name
+        from simc_support.game_data.Conduit import get_conduits_for_spec
+    except ImportError:
+        print("Error importing simc-support library for covenant test. Skipping test.")
         return
```

Comment:
Now that we have 4 places doing basically the same thing, I think we can move the import attempt of `get_wow_spec_from_combined_simc_name` (as it's used everywhere) to the top and add a bool like HAVE_SIMC_SUPPORT to check if we can run the tests.

Smth like
```python
try:
    from simc_support.game_data.WowSpec import get_wow_spec_from_combined_simc_name
    HAVE_SIMC_SUPPORT = True
except ImportError:
    HAVE_SIMC_SUPPORT = False
```
at the top and then
```python
if not HAVE_SIMC_SUPPORT:
    print("Error importing simc-support library for covenant test. Skipping test.")
    return
from simc_support.game_data.Conduit import get_conduits_for_spec
```
in this case for example.

Reply:
Or, tbh, while I don't really like external deps, I think we can just require `simc_support` at this point.
Will also let us print human readable names for talent tests.